<launch>
 <!--start_tracking.py launch file-->
 <node pkg = "my_yolo_track" type="start_tracking_yolo.py" name ="start_tracking" output="screen">
    <!--<param name="scale_linear" value="0.5" type="double"/>
    <param name="scale_angular" value="1.5" type="double"/>-->
   <remap from="start_tracking/track_cmd_vel" to="/cmd_vel_mux/input/teleop"/>
 </node>
</launch>
